src/DynamoCore/Interfaces/IPackage.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Dynamo.Interfaces
+{
+    /// <summary>
+    /// Interface that contains basic properties of a Dynamo package
+    /// </summary>
+    public interface IPackage
+    {
+        /// <summary>
+        /// Name of the package
+        /// </summary>
+        string Name { get; set; }
+
+        /// <summary>
+        /// Version of the package
+        /// </summary>
+        string VersionName { get; set; }
```

Comment:
version name?

Reply:
That's what the `Package` type calls its version property. Seemed weird to me too.

Reply:
🤦‍♂

Reply:
is it ever a name? Or is it always just a version string?

Reply:
As far as I can tell it's always the version